contracts/deployables/modules/AzoriusV1.sol

```diff
@@ -269,22 +271,37 @@ contract AzoriusV1 is IAzoriusV1, GuardableModule, Version, UUPSUpgradeable {
         address _startAddress,
         uint256 _count
     ) external view returns (address[] memory _strategies, address _next) {
+        if (
+            _startAddress != SENTINEL_STRATEGY &&
```

Comment:
this check has been included in function `isStrategyEnabled`, is it necessary to write this again?
```solidity
function isStrategyEnabled(address _strategy) public view returns (bool) {
        return
            SENTINEL_STRATEGY != _strategy &&
            strategies[_strategy] != address(0);
    }
    ```